docs/src/pages/components/dialog.mdx

```diff
@@ -0,0 +1,373 @@
+---
+name: dialog
+route: /components/dialog
+---
+
+import Button from '@pluralsight/ps-design-system-button'
+import { layout } from '@pluralsight/ps-design-system-core'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import * as Text from '@pluralsight/ps-design-system-text'
+
+import { Guideline, Intro, Types, Usage } from '../../components/component'
+import { ContentGridVisual, ModalGuidelineExample } from '../../components/dialog'
+
+# Dialog
+
+<Intro>
+  The purpose of a Dialog is to provide actionable messaging and may appear contextually or as a [modal](#modal). The Dialog adapts to various amounts and types of content. For contextual, non-actionable messaging, consider the [Tooltip](/components/tooltip) instead.
+</Intro>
+
+## Examples
+
+### In-app example
+
+Dialogs can appear automatically, or be triggered by hover, focus, tap or click.
+      
+```typescript 
+import Button from '@pluralsight/ps-design-system-button'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import { Below } from '@pluralsight/ps-design-system-position'
+import React from 'react'
+
+function Example() {
+  const [isHovered, setHovered] = React.useState(false)
+  const [isClicked, setClicked] = React.useState(false)
+  return <div className="example-grid--col-3" style={{ height: '150px' }}>
+    <Below
+      show={
+        <Dialog tailPosition={Dialog.tailPositions.topCenter}>
+          Dialog
+        </Dialog>
+      }
+      when
+    >
+      <Button appearance={Button.appearances.secondary}>
+        Look at me
+      </Button>
+    </Below>
+
+    <Below
+      show={
+        <Dialog tailPosition={Dialog.tailPositions.topCenter}>
+          Dialog
+        </Dialog>
+      }
+      when={isHovered}
+    >
+      <Button
+        appearance={Button.appearances.secondary}
+        onMouseEnter={_ => setHovered(true)}
+        onMouseOut={_ => setHovered(false)}
+      >
+        Hover me
+      </Button>
+    </Below>
+
+    <Below
+      show={
+        <Dialog tailPosition={Dialog.tailPositions.topCenter}>
+          Dialog
+        </Dialog>
+      }
+      when={isClicked}
+    >
+      <Button
+        appearance={Button.appearances.secondary}
+        onClick={_ => setClicked(!isClicked)}
+      >
+        Click me
+      </Button>
+    </Below>
+  </div>
+}
+
+export default Example
+```
+
+### Tail
+
+Dialogs can be shown with or without a tail (a directional indicator).  To make the tail appear, use a `Dialog.tailPositions` option.
+
+```typescript 
+import Button from '@pluralsight/ps-design-system-button'
+import Dialog from '@pluralsight/ps-design-system-dialog'
+import { P } from '@pluralsight/ps-design-system-text'
+import React from 'react'
+
+function Example() {
+  return <div className="example-grid--col-2">
+    <Dialog disableFocusOnMount><ExampleContent /></Dialog>
+    <Dialog tailPosition={Dialog.tailPositions.topCenter} disableFocusOnMount><ExampleContent /></Dialog>
```

Comment:
you can't really see the tail on light theme...probably not a big deal

Reply:
No, you can not.  I'm not sure what to do about that. Thought about locking dark theme. Decided against it, since that may mask a design issue in the Dialog instead of expose it.